app/webpacker/styles/_print.scss

```diff
@@ -14,12 +15,12 @@
   .govuk-tabs .govuk-tabs__title,
   .govuk-table__header.alternative-content,
   .govuk-table__cell.alternative-content,
-  #downloads.govuk-tabs__panel
-  {
+  #downloads.govuk-tabs__panel {
```

Comment:
This is ok for now but would it be worth adding a reusable `do-not-print` class?